src/database/combiningrules/outplace.jl

```diff
@@ -20,7 +20,7 @@ Ignores non-diagonal entries already set.
 If a Single Parameter is passed as input, it will be converted to a Pair Parameter with `pᵢᵢ = pᵢ`.
 """
 function kij_mix(f::F,P::SingleOrPair,K = nothing) where F
-    param = PairParam(P.name,P.components,float.(P.values),P.ismissingvalues,P.sourcecsvs,P.sources)
+    param = PairParam(P.name,P.components,P.values,P.ismissingvalues,P.sourcecsvs,P.sources)
```

Comment:
This change was done because sometimes, a parameter has an integer element type, but the mixing results in a float type (resulting in errors when doing `A[i,i] = aij`)